.github/workflows/dev.yml

```diff
@@ -15,6 +15,8 @@ jobs:
       AWS_REGION: us-east-1
       ELASTICBEANSTALK_APPLICATION_NAME: signs-ui
       ELASTICBEANSTALK_ENVIRONMENT_NAME: signs-ui-dev
+      ERL_COOKIE: ${{ secrets.ERL_COOKIE }}
```

Comment:
Per [a comment I received](https://github.com/mbta/gtfs_creator/pull/1254#discussion_r657366309) on my GHA PR, maybe move these ENV vars to the `build-push-ecr` step? And same for prod.